### DISCUSSION TITLE
forums

### DISCUSSION THREAD

**Discussion by thaliel** [easylist/easylist#17815] [General]
what has happened to the forums? I have been getting an empty page for quite a while now, and edge even claims the page is not available

**Comment by iam-py-test** [comment]
It loads fine for me. [What DNS server are you using?](https://www.howtogeek.com/786911/how-to-change-your-dns-server-on-windows-10/)
Thanks

**Reply by thaliel** [reply]
I have been using the same DNS sever ever since I set up my laptop, and it is working fine. 
I can open the forums on chrome and mobile firefox as well, but get an empty page on mobile Firefox and a "page could not be loaded" message on mobile Chrome . This is really odd

**Reply by iam-py-test** [reply]
That's bizarre.
Are they all connected to the same WiFi/Router (doubt it for the phone)? That's the only thing I can think of.
Thanks

**Reply by thaliel** [reply]
All devices use the same wifi

**Reply by iam-py-test** [reply]
Can you try restarting the router (unplugging it and plugging it back in)?
I suspect it's the cause (but no idea).
Thanks

**Reply by thaliel** [reply]
I deactivated wifi on my phone to test it via mobile data, but I get the same results. The forum front page opens (in Chrome at least), but when I try to log in or view any subforum, I get a 500 error

**Reply by iam-py-test** [reply]
Weird. No idea.
I have asked to see if anyone knows what's going on.
Thanks

**Reply by thaliel** [reply]
If I had lanik's contact info, I would send him an email asking if anything is wrong on the forums' end, but I don't.
Could you register an account to check if logging in works for you?

**Reply by mapx-** [reply]
no issue on my side

**Reply by iam-py-test** [reply]
I also checked logging in, and everything works fine for me too.
Thanks

**Reply by LanikSJ** [reply]
No problem on my end either.

**Reply by thaliel** [reply]
oh hi, thanks for replying. It is good to know this issue is only on my end. I just happened to get a new laptop, and the main page is as far as I get. Once I put in my login data and click on login, I get a 500 error again

**Reply by LanikSJ** [reply]
Please post the error.

**Reply by thaliel** [reply]
HTTP ERROR 500

**Reply by LanikSJ** [reply]
If that was coming from the forum you'd see more then that.

**Reply by thaliel** [reply]
yeah, the front page wouldn't be loading either if there was something wrong with the forums itself. This really beats me

**Reply by LanikSJ** [reply]
Clear your cache cookies etc or try a different browser.

**Reply by thaliel** [reply]
I have tried desktop firefox, mobile firefox, edge and mobile chrome, all with the same result. Hence, the issue is not related to any browser

**Reply by LanikSJ** [reply]
I see logins from earlier today on my site, so you're getting logged in.

**Reply by thaliel** [reply]
hm, so that works, I am simply not seeing it due to the 500 errors

**Comment by peace2000** [comment]
https://forums.lanik.us/

Loads fine

**Comment by getcheffy** [comment]
I'm new to all this and dumb

how do I update or add this code to ublock? I did what I could in the settings to  update everything. ij ust don't know what I'm supposed to gleen from this git page and how to use whatever may be on here, that goes for any git site without a downloadable tool/app directly

**Reply by iam-py-test** [reply]
Firstly, please do not post irrelevant comments on old threads. If you have a question, open a new thread.
Secondly, you do not need to add EasyList and EasyPrivacy to uBlock Origin; they are included by default (under the _Ads_ and _Privacy_ sections).
That said, if you wish to subscribe to a filterlist not included in uBo, [the uBlock Origin Wiki may prove helpful](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#3rd-party-filter-lists).
Thanks
